src/js/modules/getSiblings.js

```diff
@@ -0,0 +1,17 @@
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
> * [ ]  OPTIONAL
>   You have **disabled ESLint checks** on parts of your code. Please enable **all** ESLint checks on your code and try modifying that portion of your code to adhere to ESLint standards. This will ensure a robust and clean code.

I disabled it because it won't let me use `export` only for function export. It insists I use `default export` which the eslinter also rejects for function